tests/test_getlangs.py

```diff
@@ -209,24 +222,6 @@ def test_unknown_name():
 
 def test_language_names_with_modifier_in_bracket():
     # try to match based on language name (stuff before subcode in brackets)
-    lang_obj = languages.getlang_by_native_name('中文')
-    assert lang_obj is not None, 'Chinese 1 not found'
-    assert lang_obj.code == "zh", 'Wrong internal repr. code'
-    assert lang_obj.name == "Chinese", 'Wrong name'
-    assert lang_obj.native_name == "中文 (Zhōngwén), 汉语, 漢語", 'Wrong native_name'
-    #
-    lang_obj = languages.getlang_by_native_name('汉语')
-    assert lang_obj is not None, 'Chinese 2 not found'
-    assert lang_obj.code == "zh", 'Wrong internal repr. code'
-    assert lang_obj.name == "Chinese", 'Wrong name'
-    assert lang_obj.native_name == "中文 (Zhōngwén), 汉语, 漢語", 'Wrong native_name'
-    #
-    lang_obj = languages.getlang_by_native_name('漢語')
-    assert lang_obj is not None, 'Chinese 3 not found'
-    assert lang_obj.code == "zh", 'Wrong internal repr. code'
-    assert lang_obj.name == "Chinese", 'Wrong name'
-    assert lang_obj.native_name == "中文 (Zhōngwén), 汉语, 漢語", 'Wrong native_name'
-    #
```

Comment:
These test were removed because the non-standard `(Zhōngwén)` string was removed from the `native_name`